Gopkg.lock

```diff
@@ -699,7 +698,6 @@
     "github.com/containernetworking/plugins/pkg/ns",
     "github.com/dlespiau/covertool/pkg/cover",
     "github.com/docker/go-units",
-    "github.com/firecracker-microvm/firecracker-go-sdk",
```

Comment:
uhmm I wonder why this was removed

cc @mcastelino

Reply:
@nitkon did this get removed automatically. Is it possible the vendoring tool is not picking up the firecracker dependency properly.

We do use the swagger generated files from the firecracker-go-sdk

Reply:
Ok. I think I have an explanation for this but still need to test this PR.

In fc.go we only use the generated files

```
        "github.com/firecracker-microvm/firecracker-go-sdk/client"
        models "github.com/firecracker-microvm/firecracker-go-sdk/client/models"
        ops "github.com/firecracker-microvm/firecracker-go-sdk/client/operations"
```

The files that are getting deleted are under github.com/firecracker-microvm/firecracker-go-sdk, which I think are not used by fc.go or its dependencies. (I will verify).

Somehow the vendoring tool did not do the right thing when I vendored the code and is now doing the right set of files. 

Let me retest and confirm.

Reply:
@nitkon verified. So LGTM